### PR TITLE
Readme update for testing.

### PR DESCRIPTION
**Pull request by christhompsongoogle**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,11 +82,17 @@ See the section about [running tests](https://facebook.github.io/create-react-ap
 To run the test runner with emulators, use:
 
 ```bash
-firebase emulators:exec --project sample --only firestore 'npm test'
```
Comment (yuchenshi) [inline README.md:85]: Why this change?

**Review by yuchenshi — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (christhompsongoogle) [reply README.md:85]: Whoops, missed this one: basically as an easier copy/paster since we have the --only demonstrated below
